bundles/paikkatietoikkuna/coordinatetransformation/components/CoordinateTable.js

```diff
@@ -156,14 +194,14 @@ Oskari.clazz.define('Oskari.coordinatetransformation.component.table', function(
         setIsEditable: function ( editable ) {
             this.isEditable = editable;
             var rows = this.getElements().rows;
-            if( !editable ) {
+            if( editable ) {
```

Comment:
You could remove the if/else-statement and just do:

    rows.each( function () {
        jQuery(this).find('.cellContent').attr("contenteditable", editable);
    });